docs/source-pytorch/common/trainer.rst

```diff
@@ -1479,7 +1479,8 @@ How often within one training epoch to check the validation set.
 Can specify as float or int.
 
 - pass a ``float`` in the range [0.0, 1.0] to check after a fraction of the training epoch.
-- pass an ``int`` to check after a fixed number of training batches.
+- pass an ``int`` to check after a fixed number of training batches. An ``int`` value can only be higher than the number of training
+  batches when ``check_val_every_n_epoch=None``, which validates after every ``N`` training batches across ``trainer.fit``.
```

Comment:
IMO this is clearer

```suggestion
  batches when ``check_val_every_n_epoch=None``, which validates after every ``N`` training batches across epochs.
```

Reply:
what if epochs=None and the user is performing iteration-based training? It might be confusing then IMO